SparkPost/2web2ui
# TR-2223: Update AMP validator script to latest version

### What Changed
 - Upgraded the AMP Validator script

### How To Test
 - Confirm no linting errors are present when inserting AMP Boilerplate
 - Confirm a couple of AMP examples https://amp.dev/documentation/examples/ render without linting errors
- Confirm empty AMP content provided errors
